scripts/sct_analyze_lesion.py

```diff
@@ -550,6 +538,11 @@ def main(args=None):
     verbose = arguments.v
     sct.init_sct(log_level=verbose, update=True)  # Update log level
 
+    # TODO: CG: I would like to replace AnalyzeLesion by two classes:
+    #               - LesionVolume: which contains all the lesions, an ID assigned to each individual lesion, and
+    #                               and computing / gathering overall (ie across all lesions) metrics
+    #               - Lesion: which is dedicated to one individual lesion and computes / gathers metrics specific to
+    #                           this lesion.
```

Comment:
@jcohenadad: What do you think? :-)

Reply:
it's hard to tell without seeing how these classes will interact with the `aggregate_metrics` module. I am also wondering if a class wouldn't be an overkill, given that its scope would be limited (only used here), and a dict could potentially achieve the same. It would be great to have @kousu input for overall design of the function